.vortex/tooling/src/helpers.php

```
@@ -549,6 +644,116 @@ function lagoon_exec(string $bin, string $subcommand, array $ctx, ?int &$exit_co
+  }
+
+  ob_start();
+  passthru($cmd, $exit_code);
+  $output = ob_get_clean();
+
+  if (!$exit_code_provided && $exit_code !== 0) {
+    fail('Acquia CLI command "%s" failed with exit code %s. Output: %s', $subcommand, $exit_code, $output);
+  }
+
+  return $output === FALSE ? '' : $output;
+}
```

PHPMD (2.15.0)

[error] The parameter $exit_code is not named in camelCase. (undefined)

(CamelCaseParameterName)